ui/src/SoundWaveCanvas.js

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+import {SoundWave} from './SoundWave';
+import {LitElement, html, css} from 'lit-element';
+
+export class SoundWaveCanvas extends LitElement {
+    static get properties() {
+        return {
+            canvasId: {type: String},
+            canvas: {type: Object},
+            audioStream: {type: Object},
+            isRecording: {type: Boolean}
+        };
+    }
+    constructor() {
+        super();
+        this.soundWave;
+        this.canvasId = "myCanvas";
+        this.canvas;
+    }
+    render() {  
+        return html`
+            <canvas id=${this.canvasId} width="400" height="400"></canvas> 
```

Comment:
nit: make a CSS class for height/width

Reply:
you can attach CSS pretty easily https://lit-element.polymer-project.org/guide/styles

in this case it'd probably look like

```js
    static get styles() {
        return css`
            canvas {
                width: 400px;
                height: 400px;
            }
        `;
      }
```
but I'm not sure if canvas does width and height this way; you'd probably have to do some further research

Reply:
I recall canvas being very finicky and needing height, width defined on the element itself. See if it works or not?

Also if you need variable width and need to set this field on the html, you can do this with lit-html

```html
<canvas id=${this.canvasId} width="${myWidth}" height="${myHeight}">
```

Reply:
I was just going to ask about that. Changing where width and height was defined made the soundwave look wonky. Guess im keeping it on the tag.